students/erica_edwards/Lesson03/Lesson3_Slicing_Lab.py

```diff
@@ -0,0 +1,47 @@
+def exchange_first_last(sequence):
+    a = sequence[-1:]  
+    b = sequence[1:-1]  
+    c = sequence[0:1]
+    result = a + b + c
+    return result
+
+def every_other_item(sequence):
+    result = sequence[0:-1:2]
+    return result
+
+def first_last_middle(sequence):
+    x = int(len(sequence)/3)
+    result = sequence[x:-x:2]
+    return result
```

Comment:
there should be one with "With the first and last 4 items removed, and every other item in between" ?